src/memory/new_delete.cpp

```diff
@@ -41,9 +41,7 @@ void *newImpl(const std::size_t size, const std::align_val_t align) {
 }
 
 void *newNoExcept(const std::size_t size) noexcept { return malloc(size); }
-void *newNoExcept(const std::size_t size, const std::align_val_t align) noexcept {
-  return aligned_alloc(size, static_cast<std::size_t>(align));
-}
+void *newNoExcept(const std::size_t size, const std::align_val_t align) noexcept { return malloc(size); }
```

Comment:
DANGER: why `aligned_alloc` -> `malloc`

Reply:
yes yes, reverted that, it was just some mine testing